scripts/stackstorm.js

```diff
@@ -45,7 +45,7 @@ var _ = require('lodash'),
   ;
 
 // Setup the Environment
-env.ST2_API = env.ST2_API || 'http://localhost:9101';
+env.ST2_API_URL = env.ST2_API_URL || env.ST2_API || 'http://localhost:9101';
```

Comment:
To ensure backwards compatibility, I would do this:

```js
env.ST2_API_URL = env.ST2_API_URL || null;

if (!env.ST2_API_URL) {
  if (env.ST2_API) {
    robot.logger.warning("ST2_API is now deprecated and will be removed in a future release. Instead, please use the ST2_API_URL environment variable (the value can remain the same).");
  }
  env.ST2_API_URL = env.ST2_API || 'http://localhost:9101';
}
```

Reply:
That's a good addition, works for me :+1: